docusaurus.config.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = {
+  title: 'OpenHIM',
+  tagline: 'Simplifying Interoperability',
+  url: 'https://jembi.github.io',
```

Comment:
@BMartinos Why are there go carts racing at this url?

Reply:
@MattyJ007 i think this is something that has been there for quite some time without anybody noticing :/. This was a jembi event that happened about 5 years ago. That repo must have been pushed to the jembi github and not to a sub-directory